### ISSUE TITLE
Command \iftikzmark ignores suffix

### ISSUE THREAD

**Issue by paternal** [loopspace/tikzmark#2]
Here is a MWE:

~~~tex
\documentclass{article}

\usepackage{tikz}
\usetikzlibrary{tikzmark}

\begin{document}

% Without suffix

\tikzmark{bar}
\iftikzmark{bar}{
  TIKZMARK BAR IS AVAILABLE
}{
  TIKZMARK BAR IS NOT AVAILABLE
}

% With suffix

\tikzset{tikzmark suffix=-foo}%

\tikzmark{baz}
\iftikzmark{baz}{
  TIKZMARK BAZ IS AVAILABLE
}{
  TIKZMARK BAZ IS NOT AVAILABLE
}

\end{document}
~~~

- What I expect: a document displaying `TIKZMARK BAR IS AVAILABLE` and `TIKZMARK BAZ IS AVAILABLE`.
- What I get: a document displaying `TIKZMARK BAR IS AVAILABLE` (good) and `TIKZMARK BAZ IS NOT AVAILABLE` (bad).

-- Louis

**Comment by loopspace** [comment]
Apologies for the delay in responding.

Absolutely right.  Fixed (I believe) in https://github.com/loopspace/tikzmark/commit/909d18b78263ea1f78b8397882230c915632f470 (please test!)

**Comment by paternal** [comment]
> please test!

Looks fine to me, but since I reported this issue six months ago, I no longer use this package (I stopped working on the project I used tikzmark for, since it was a bad answer to a badly stated problem). So I am looking at this issue with a naïve eye, and I might be missing obvious things…

Thanks for your work!

-- Louis

**Comment by paternal** [comment]
Had a look at this issue. Indeed, seems closed by 909d18b.
Thanks
